meetings/2021-08-18.md

```diff
@@ -0,0 +1,181 @@
+# 2021-08-18 Solid Authorization
+
+https://meet.jit.si/solid-authorization
+
+
+## Agenda
+
+* Minutes 2021-08-11 - https://github.com/solid/authorization-panel/pull/246
+* [PR 244: Effective ACR discovery](https://github.com/solid/authorization-panel/pull/244)
+* [PR 249: Added first delegation use case](https://github.com/solid/authorization-panel/pull/249)
+* [PR 241: Fixes and add story for privacy](https://github.com/solid/authorization-panel/pull/241)
+* Access Modes currently used by: WAC, ACP (5min)
+* Reliance on client interpreting IRIs (aka. `/` hierarchy): WAC, ACP (5min)
+
+
+## Present
+
+* Barath
+* Elf
+* Henry
+* Matthieu
+* Justin
+
+
+## Minutes
+
+### Effective ACR discovery
+
+Henry: Issues related to optimizing the number of requests for effective ACR discovery:
+- [Support Trig serialization of Access Control Resources](https://github.com/solid/authorization-panel/issues/247)
+- [Define another relation for non-created ACR](https://github.com/solid/authorization-panel/issues/248)
+I describe my favorite option in issue #248, which adds a new "control" link type to link to the potential (not-yet-created) ACR for when a default ACR is being used.
+I need to create these issues to be able to link to them in order [PR 244: Effective ACR discovery](https://github.com/solid/authorization-panel/pull/244), and so for it to be merged.
+
+Matthieu: Given that we have 4 potential solutions to minimising the number of requests involved in discovering the effective ACR, is everyone ok to merge #244?
+
+Henry: Yes.
+
+Elf: +1
+
+Justin: +1
+
+Matthieu: Merged.
+
+Henry: The last solution might be useful for ACP too, if you want to weaken the requirement that every resource must have one ACR.
+
+Matthieu: I'm not sure that falls into the ACP model, but let's review once the draft ACP PR is ready for review (that will be done today).
+
+
+### Delegation use case
+
+Henry: Could you give us a review of that use case, Elf?
+
+Elf: How do you feel about using the [GNAP terminology](https://www.ietf.org/archive/id/draft-ietf-gnap-core-protocol-00.html#name-roles) of Resource owner, server...?
+
+Justin: I like using that terminology because I think it's well thought out and aligns with where we're going in terms of different actors and roles and responsibilities.
+
+Henry: Btw, a bit out of context, but how long is the GNAP spec since we mentioned it (I think it was 80 pages last time I looked)?
```

Comment:
```suggestion
Henry: Btw, a bit out of context, but how long is the GNAP spec since we mentioned it? I think it was 80 pages last time I looked...
```